roles/opensrp/tasks/main.yml

```diff
@@ -63,10 +63,26 @@
     owner: "{{ opensrp_system_user }}"
     group: "{{ tomcat_group }}"
 
+- name: Download CA certificate bundle
+  become: yes
+  become_user: root
+  get_url:
+    url: "{{ ca_bundle_url }}"
```

Comment:
@ndegwamartin This should follow the naming convention used in the roles. So maybe `opensrp_ca_bundle_url`.

Reply:
This is updated too